### DISCUSSION TITLE
Award BIOS logo format is not standard BMP

### DISCUSSION THREAD

**Discussion by antoniu200** [platomav/BIOSUtilities#42] [General]
EDIT: Convert to .tiff using [AWBMTools](https://www.blackfiveservices.co.uk/awbmtools.shtml).

~Hello,~

~I am trying to extract files from the latest Award BIOS versions for the H55M-UD2H and EX58-UD3R, but no files can be opened from either extraction. Also, total size of extracted files is lower than the original compressed BIOS file.~
~Using Python 3.11.0, dissect.util 3.15, latest pefile, latest 64 bit 7-Zip, Windows 11 22H2.1485.~

Not sure what is wrong here.

Any advice?

Thanks!

**Comment by platomav** [comment]
I don't see any problem:

![Screenshot 2024-05-06 110627](https://github.com/platomav/BIOSUtilities/assets/11527726/88dae993-e0aa-4322-8d74-c801f9e63dba)

Only 7-Zip is required for the Award utility. Make sure you have "7z.exe" copied to the "external" directory.

**Comment by antoniu200** [comment]
I do have 7z.exe in the "external" folder and I get all the command-line output in your screenshot, but the extracted files are corrupt and I can't open anything.

Easiest to check on the BMP logos. Are they fine on your end?

I'll zip the output I get after running your script and send it to you when I get to my PC (no idea why I haven't done this in the first place).

**Comment by platomav** [comment]
The files are correct, everything looks good when previewed with a hex editor. The Award "bmp" files are not the normal "BMP" format images. Found this with a quick google search:

- http://fileformats.archiveteam.org/wiki/Award_BIOS_logo

**Reply by antoniu200** [reply]
Whoops... Oh well, at least I know from now on.

Thanks! And sorry if I was a nuisance. I will change the title now to be more helpful for others in the future.

**Reply by platomav** [reply]
No worries! 👍
